ChangeLog.d/fix_cmake_gen_files.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Fix an issue with in-tree CMake builds in releases with GEN_FILES
+     turned off: if a shipped file was missing from the working directory,
+     it could be turned into a symbolic link to itself.
```

Comment:
I still don't understand this.

Is it "if a generated file wasn't present in the working directory, a symbolic link pointing to itself would be created"?

Reply:
That's how I have parsed it, yes.

Reply:
That is my understanding of https://github.com/Mbed-TLS/mbedtls/pull/6086 indeed. Possibly only under some circumstances (I haven't investigated), hence “could”.

Reply:
I think Tom's point is that if it's shipped, it's not missing? You could say "if a file that should have been shipped was missing..." but it's getting a bit wordy? IMO the current wording isn't optimal but it's sufficient.

Reply:
What can happen in practice is that a file was shipped, but is not present at this time (because the user removed it, e.g. by running `make neat`). The file would not be “omitted” for anywhere, but it would be “missing” from the build tree.

Reply:
that's why I used "not present", which covers all of those things, and means just that's it's not there